src/openforms/js/components/formio_builder/translation.js

```diff
@@ -124,7 +124,16 @@ export const extractComponentLiterals = component => {
     ...ALWAYS_TRANSLATABLE_PROPERTIES,
     ...(ADDITIONAL_PROPERTIES_BY_COMPONENT_TYPE[component.type] || []),
   ];
-  const literals = allTranslatableProperties
+
+  // Do not show warnings for date/datetime placeholders, they are not translatable
+  const isDateOrDatetimeComponent = component.type === 'date' || component.type === 'datetime';
+  const dateOrDatetimeTranslatableProperties = allTranslatableProperties.filter(
+    property => property !== 'placeholder'
+  );
+
+  const literals = (
+    isDateOrDatetimeComponent ? dateOrDatetimeTranslatableProperties : allTranslatableProperties
+  )
     .filter(property => !isRegex(property))
     .map(property => get(component, property));
```

Comment:
Maybe something like:

```suggestion
  const isDateOrDatetimeComponent = component.type === 'date' || component.type === 'datetime';

  const literals = allTranslatableProperties
    // Do not show warnings for date/datetime placeholders, they are not translatable:
    .filter(property => isDateOrDatetimeComponent ? property !== 'placeholder' : property)
    .filter(property => !isRegex(property))
    .map(property => get(component, property));
```

Reply:
I like it, I'll apply it and make sure this PR gets merged.